nixos/common/default.nix

```diff
@@ -21,6 +21,7 @@
   # - for containers we currently rely on the `stage-2` init script that sets up our /etc
   boot.initrd.systemd.enable = lib.mkDefault (
     !config.boot.initrd.network.enable &&
+    !(if lib.versionOlder (lib.versions.majorMinor lib.version) "23.11" then config.boot.initrd.services.swraid.enable else config.boot.swraid.enable) &&
     !config.boot.swraid.enable &&
```

Comment:
Didn't actually replace the problematic line

Reply:
Yeah. This was done later here: https://github.com/numtide/srvos/commit/6da84e4e1f742d0590c6b25712ecc1adc22f42b8